packages/metal-quartz-checkbox/src/MetalQuartzCheckbox.js

```diff
@@ -19,6 +19,53 @@ class MetalQuartzCheckbox extends Component {
  * @type {!Object}
  */
 MetalQuartzCheckbox.STATE = {
+	checkboxInline: {
```

Comment:
Yes that is what it is, should look like when initializing from js:
```
checkboxInline: [{
	checked: true,
	label: 'Inline 1'
}, {
	label: 'Inline 2'
}, {
	checked: true,
	disabled: true,
	label: 'Inline 3'
}, {
	label: 'Inline 4'
}, {
	label: 'Inline 5'
}, {
	label: 'Inline 6'
}]
```
I was following what was being done at https://github.com/liferay-lexicon/lexicon-dropdown/blob/master/demos/index.html